models/openacademy_session.py

```diff
@@ -1,12 +1,48 @@
-from odoo import models, fields
+from odoo import models, fields, api
+from odoo.exceptions import ValidationError
 
 class Session(models.Model):
     _name = "openacademy.session"
     _description = "This is the description of Session model."
 
     name = fields.Char()
-    start_date = fields.Date()
+    start_date = fields.Date(default=fields.Date.today)
     duration = fields.Integer()
     number_seats = fields.Integer("Number of seats")
-    instructor_id = fields.Many2one("res.partner", ondelete="restrict")
+    instructor_id = fields.Many2one("res.partner", domain="['|', ('instructor', '=', 'True'), ('category_id.name', '=', 'Teacher / Level 1'), ('category_id.name', '=', 'Teacher / Level 2')]", ondelete="restrict")
     course_id = fields.Many2one("openacademy.course", ondelete="restrict", required=True)
+    attendee_ids = fields.Many2many("res.partner", string="Attendees")
```

Comment:
provide relational table and columns. Not only is a good practice, but also if you don't do it, you won't be able to create the inverse field on partners correctly (partner's field will be a new one).

Reply:
I see you provided attributes correctly. However, you need to provide them also on the inverse field, using the same table but columns inverted. That's the way you could ensure both fields use the same m2m relation.